### PR TITLE
[Fixes # 5277] Formatting refresh for New-LocalUser

### LINKED ISSUE #5277: New-LocalUser Refresh
# Documentation Issue
All Versions of New-LocalUser documentation needs to be refreshed for current MD formatting

## Context of the issue

- URL to the article (Docs or GitHub URL): https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.localaccounts/new-localuser?view=powershell-5.1
- Issue affects the following content (check all that apply):
  
  **Cmdlet reference & about_ topics**
  - [x] Version 7 content
  - [x] Version 6 content
  - [x] Version 5.1 content

  **Conceptual articles**
  - [ ] Fundamental conceptual articles
  - [ ] Script sample articles
  - [ ] DSC articles
  - [ ] Gallery articles
  - [ ] JEA articles
  - [ ] WMF articles
  - [ ] SDK articles
- Is the issue specific to a platform (Y/N - Win/macOS/Linux):

### PR DESCRIPTION
**Pull request by chasewilson**
# PR Summary
Completes Formatting refresh for New-LocalUser Doc

## PR Context
Fixes #5277 
Fixes [AB#1661835](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1661835)

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [ ] Version 7 content
- [ ] Version 6 content
- [x] Version 5.1 content

**Conceptual articles**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

File: reference/5.1/Microsoft.PowerShell.LocalAccounts/New-LocalUser.md [hunk 1]
```diff
@@ -12,41 +12,48 @@ title: New-LocalUser
...
+
 ```
 PS C:\> New-LocalUser -Name "User02" -Description "Description of this account." -NoPassword
 Name    Enabled  Description
 ----    -------  -----------
 User02  True     Description of this account.
 ```
 
-This command creates a local user account and does not specify the *AccountExpires* or *Password* parameters. Therefore, the account doesn't expire or have a password by default.
+This command creates a local user account and does not specify the _AccountExpires_ or _Password_
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.LocalAccounts/New-LocalUser.md]: Parameter names should be bold.

File: reference/5.1/Microsoft.PowerShell.LocalAccounts/New-LocalUser.md [hunk 2]
```diff
@@ -181,10 +193,12 @@ Accept wildcard characters: False
...
 
 ### -Password
-Specifies a password for the user account.
-You can use `Read-Host -GetCredential`, Get-Credential, or ConvertTo-SecureString to create a **SecureString** object for the password.
 
-If you omit the *Password* and *NoPassword* parameters, **New-LocalUser** prompts you for the new user's password.
+Specifies a password for the user account. You can use `Read-Host -GetCredential`, `Get-Credential`,
+or `ConvertTo-SecureString` to create a **SecureString** object for the password.
+
+If you omit the _Password_ and _NoPassword_ parameters, `New-LocalUser` prompts you for the new
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.LocalAccounts/New-LocalUser.md]: Bold parameters

File: reference/5.1/Microsoft.PowerShell.LocalAccounts/New-LocalUser.md (hunk 1 above)
Comment (chasewilson) [reply reference/5.1/Microsoft.PowerShell.LocalAccounts/New-LocalUser.md]: fixed

File: reference/5.1/Microsoft.PowerShell.LocalAccounts/New-LocalUser.md (hunk 2 above)
Comment (chasewilson) [reply reference/5.1/Microsoft.PowerShell.LocalAccounts/New-LocalUser.md]: fixed

**Pull request merged**